### DISCUSSION TITLE
Question about units

### DISCUSSION THREAD

**Discussion by ZacharyEBirch** [astra-toolbox/astra-toolbox#433] [Q&A]
I am working with a set of CBCT data and am using the sinogram of the center slice and running it through the fanbeam reconstruction module.

Specifications:
Projection Angles: 1000 (0-360 Degree sweep)
Row Pixels: 4000
Pixel Pitch: 0.008mm
SOD: 65mm
SDD: 260mm

I've noticed that if I switch the SOD, SDD and pixel pitch to microns I get different results, and was wondering why this was the case? 
Working in millimeters actually gives me nonsense in this case in the reconstruction, while the reconstruction in microns is zoomed in.

**Comment by askorikov** [comment]
Hi! I think it may be related to the voxel size of the reconstruction. When you create a volume geometry with `astra.create_vol_geom(Y, X, Z)` a voxel size of (1.0. 1.0, 1.0) [arbitrary units] is assumed, whereas you probably want it to be similar to the pixel size of the detector. To set a different voxel size, you can use `astra.create_vol_geom(Y, X, Z, minx, maxx, miny, maxy, minz, maxz)` syntax, where min/max parameters control the extent of the volume and therefore the voxel size.

**Reply by ZacharyEBirch** [reply]
This was exactly it, thanks so much.

**Reply by askorikov** [reply]
Glad to help!
